ansible/roles/enroot/defaults/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+enroot_ver: 3.4.1
+enroot_pkg_ver: 1
+enroot_deb: enroot_{{ enroot_ver }}-{{ enroot_pkg_ver }}_{{ deb_architecture.stdout }}.deb
```

Comment:
replace `deb_architecture.stdout` with `deb_arch` as described below.

Reply:
I think you can also support the hardened debs/rpms relatively easily with:

```
enroot_hardened: false
root_deb: enroot{{ "-hardened" if enroot_hardened else "" }}_{{ enroot_ver }}-{{ enroot_pkg_ver }}_{{ deb_arch }}.deb
```

Reply:
Done - modified it a bit to make more sense.  For some reason the rpm links use a dash in a location that the deb files use an underscore.